ClientMapLockTest.testLockTTLExpires_usingIsLocked

```
java.lang.AssertionError: null
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertFalse(Assert.java:64)
	at org.junit.Assert.assertFalse(Assert.java:74)
	at com.hazelcast.client.map.ClientMapLockTest.testLockTTLExpires_usingIsLocked(ClientMapLockTest.java:429)
```

https://hazelcast-l337.ci.cloudbees.com/view/Official%20Builds/job/Hazelcast-3.maintenance-IbmJDK1.8/com.hazelcast$hazelcast-client-legacy/50/testReport/junit/com.hazelcast.client.map/ClientMapLockTest/testLockTTLExpires_usingIsLocked/

--- a/hazelcast-client/src/test/java/com/hazelcast/client/map/ClientMapLockTest.java
+++ b/hazelcast-client/src/test/java/com/hazelcast/client/map/ClientMapLockTest.java
@@ -419,14 +419,14 @@ public class ClientMapLockTest {
     public void testLockTTLExpires_usingIsLocked() throws Exception {
         final IMap map = client.getMap(randomString());
         final String key = "key";
-        map.lock(key, 3, TimeUnit.SECONDS);
+        map.lock(key, 2, TimeUnit.SECONDS);
 
-        final boolean isLockedBeforeSleep = map.isLocked(key);
-        sleepSeconds(4);
-        final boolean isLockedAfterSleep = map.isLocked(key);
-
-        assertTrue(isLockedBeforeSleep);
-        assertFalse(isLockedAfterSleep);
+        assertTrueEventually(new AssertTask() {
+            @Override
+            public void run() throws Exception {
+                assertFalse(map.isLocked(key));
+            }
+        }, 10);
     }
 
     @Test

